hands-on-exercise/1-ignite-cli/4-create-message.md

```diff
@@ -212,7 +211,7 @@ import "github.com/b9lab/checkers/x/checkers/testutil"
 const (
     alice = testutil.Alice
     bob   = testutil.Bob
-    carol = testutil.Bob
+    carol = testutil.Carol
```

Comment:
👍

Reply:
Also, I've been following the tutorials but the tutorial's code somehow doesn't work. It seems because, in the first part of the tutorial, Ignite CLI prompts learners to initialize the chain under git repo github.com/alice/checkers. But the repo package changed to github.com/b9lab/checkers in the later part of the tutorials, thus making package not found errors.

Reply:
That is intentional, the tutorial content does not make any mention of b9lab, but the code itself is made to be able to be hosted at `github.com/b9lab/checkers`...
We are well aware that it causes confusion for those who switch between copy-pasting the snippets, and checking out the code sometimes.